src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -993,6 +991,12 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         const auto ptr = args.Pointer();
         const auto point = args.GetCurrentPoint(*this);
 
+        if (!_focused)

```

Comment:
This doesn’t break Leon’s work on normal click drag focus right?

Reply:
Like clicking and dragging an unfocused terminal should raise and focus _and_ begin a selection, even with panes

Reply:
Should a mouse event click get into the terminal even if it’s not focused, AND focus it? It looks like other terminals DO send the event in.

Reply:
Working with him on it now. I'm gonna try and pipe something like `IsInVTMouseMode()` up from TermInput to TermControl. That'll solve all of these problems.

The problem right now is, we just try to send a mouse event, as opposed to knowing if we are in that input mode (as opposed to selection mode). That way, we have a good story for interacting with the terminal when there's no focus.